lms/templates/dashboard.html

```diff
@@ -1,12 +1,14 @@
+<%page expression_filter="h"/>
 <%inherit file="main.html" />
 <%namespace name='static' file='static_content.html'/>
 <%!
+from django.core.urlresolvers import reverse
 from django.utils.translation import ugettext as _
 from django.template import RequestContext
+import json
```

Comment:
good catch.  I hadn't added the import; just moved it.  But it's not even used in the file.